builders/manage/sudo/.pages

```diff
@@ -0,0 +1,7 @@
+title: Using Sudo
+hide: false
+nav:
+  - index.md
+  - sudo.md
+  - upgrade.md
```

Comment:
these should follow the same pattern as the 'Minting Tokens': 'minting.md'

maybe 'Manage the/Your Sudo Account' and 'Dispatch Runtime Upgrades'